site/content/how-to/monitoring/tracing.md

```diff
@@ -13,6 +13,8 @@ NGINX Gateway Fabric supports tracing using [OpenTelemetry](https://opentelemetr
 
 This guide explains how to enable tracing on HTTPRoutes using NGINX Gateway Fabric. It uses the OpenTelemetry Collector and Jaeger to process and collect the traces.
 
+To see all the possible configurations for our resources, pleas refer to the [API reference]({{< relref "reference/api.md" >}}).
```

Comment:
```suggestion
Fo all the possible configuration options for these resources, see to the [API reference]({{< relref "reference/api.md" >}}).
```

I would actually include this down below in the Enable Tracing section after we list the required policies, instead of here.

Reply:
Also include this in the client settings doc, as well as in a `Further reading` section at the bottom of each of these docs.

Reply:
That client settings doc hasn't been merged yet, right?

Reply:
Oh you're right

Reply:
FYI: client settings doc has been merged

Reply:
Do we need to mention it twice? In a further reading section too?

Reply:
the only reason I thought it could be beneficial is if someone skimmed over the first link and went through the guide, and then at the end wanted to check out the full spec